lib/stripe/stripe_client.rb

```diff
@@ -7,21 +7,29 @@ module Stripe
   # recover both a resource a call returns as well as a response object that
   # contains information on the HTTP call.
   class StripeClient
+    include Stripe::ClientAPIOperations
+
     # A set of all known thread contexts across all threads and a mutex to
     # synchronize global access to them.
     @thread_contexts_with_connection_managers = []
     @thread_contexts_with_connection_managers_mutex = Mutex.new
     @last_connection_manager_gc = Util.monotonic_time
 
-    # Initializes a new `StripeClient`.
-    #
-    # Takes a connection manager object for backwards compatibility only, and
-    # that use is DEPRECATED.
-    def initialize(_connection_manager = nil)
```

Comment:
I wasn't sure what the plan was with this deprecated param. Considering how large of a feature this is, I'm wondering if it's acceptable to say the next version is a breaking change for those depending on the ability to pass in a connection manager?

Reply:
Yeah, I can do some introspection on `config_overrides` to ignore it if it's a connection manager object.